tests/e2e/e2e_test.go

```diff
@@ -2116,6 +2116,15 @@ func TestE2E(t *testing.T) {
 					E2ELog(t, "Tainting node failed")
 					return err
 				}
+
+				removeTaintClosure := func() {
+					err := removeNodeTaint(t, f, taintedNode.Name, taintKey)
```

Comment:
Let's get the `removeNodeTaint` to be wrapped by a retry.

Reply:
Done